### PR TITLE
AD Review

### PR DESCRIPTION
**Pull request by marco-tiloca-sics**
This PR addresses:

* The AD Review of version -16
   - https://mailarchive.ietf.org/arch/msg/ace/fRBQjDr2m4Q5kmGesUIP23ueYJo/

* The following issues:
   - https://github.com/ace-wg/ace-key-groupcomm/issues/146
   - https://github.com/ace-wg/ace-key-groupcomm/issues/147
   - https://github.com/ace-wg/ace-key-groupcomm/issues/148
   - https://github.com/ace-wg/ace-key-groupcomm/issues/149
   - https://github.com/ace-wg/ace-key-groupcomm/issues/150
   - https://github.com/ace-wg/ace-key-groupcomm/issues/151
   - https://github.com/ace-wg/ace-key-groupcomm/issues/152
   - https://github.com/ace-wg/ace-key-groupcomm/issues/153
   - https://github.com/ace-wg/ace-key-groupcomm/issues/154

### CODE REVIEW

**Review by paulwouters — APPROVED** [review]
looks good - not sure if you were waiting on my github approval so I flagged it - ignore if you are waiting for co-authors or WG :)

**Review by fpalombini — APPROVED** [review]
Looks good!

**Pull request merged**
